client/js/compute.js

```diff
@@ -0,0 +1,578 @@
+      name: '>',
+      desc: 'returns true if x is greater than y',
+      sample: 'var a = ${x} > ${y}',
+      call: function(v, x, y) { return v = x > y }
+    },
+    {
+      name: '&&',
+      desc: 'logical AND returns true if both x and y are true or returns the value of x or y if they are not boolean values',
+      sample: 'var a = ${x} >= ${y}',
+      call: function(v, x, y) { return v = x && y }
+    },
+    {
+      name: '||',
+      desc: 'logical OR returns true if either x or y are true or returns the value of x or y if they are not boolean values',
+      sample: 'var a = ${x} >= ${y}',
+      call: function(v, x, y) { return v = x || y }
+    },
+    {
+      name: '!',
+      desc: 'logical NOT returns true if the given value is false (or can be evaluated as false), otherwise returns false',
+      sample: 'var a = ! ${x}',
+      call: function(v, x) { return v = !x }
+    },
+    {
+      name: 'hypot',
+      desc: 'returns the square root of the sum of squares (hypotenuse) - a second number y is optional',
+      sample: 'var a = hypot ${x} ${y}',
+      call: function(v, x, y) { return v = (y !== undefined) ? Math.hypot(x, y) : Math.hypot(x) }
+    },
+    {
+      name: 'max',
+      desc: 'returns the larger/higher of two numbers',
+      sample: 'var a = max ${x} ${y}',
+      call: function(v, x, y) { return v = Math.max(x, y) }
+    },
+    {
+      name: 'min',
+      desc: 'returns the smaller/lower of two numbers',
+      sample: 'var a = min ${x} ${y}',
+      call: function(v, x, y) { return v = Math.min(x, y) }
+    },
+    {
+      name: 'pow',
+      desc: 'returns the base to the exponent power (x to y)',
+      sample: 'var a = pow ${x} ${y}',
+      call: function(v, x, y) { return v = Math.pow(x, y) }
+    },
+    {
+      name: 'sin',
+      desc: 'returns the sine of a number (in degrees)',
+      sample: 'var a = sin ${x}',
+      call: function(v, x) { return v = Math.sin(x * Math.PI/180) }
+    },
+    {
+      name: 'cos',
+      desc: 'returns the cosine of a number (in degrees)',
+      sample: 'var a = cos ${x}',
+      call: function(v, x) { return v = Math.cos(x * Math.PI/180) }
+    },
+    {
+	  name: 'tan',
+      desc: 'returns the tangent of a number (in degrees)',
+      sample: 'var a = tan ${x}',
+      call: function(v, x) { return v = Math.tan(x * Math.PI/180) }
+    },
+    {
+      name: 'asin',
+      desc: 'returns the arcsine (in degrees) of a number',
+      sample: 'var a = asin ${x}',
+      call: function(v, x) { return v = Math.asin(x) / Math.PI*180 }
+    },
+    {
+      name: 'acos',
+      desc: 'returns the arccosine (in degrees) of a number',
+      sample: 'var a = cos ${x}',
+      call: function(v, x) { return v = Math.acos(x) / Math.PI*180 }
+    },
+    {
+      name: 'atan',
+      desc: 'returns the arctangent (in degrees) of a number',
+      sample: 'var a = atan ${x}',
+      call: function(v, x) { return v = Math.atan(x) / Math.PI*180 }
+    },
+    {
```

Comment:
I think lines 161 and 162 should probably be:
sample: 'var a = atan2 ${y} ${x}',
call: function(v, x, y) {return v = Math.atan2(y, x) / Math.PI*180 }

In order for the x and y variables to match up with the x and y coordinates.
![image](https://user-images.githubusercontent.com/78324099/119889379-18150980-befc-11eb-9510-0549b57d27ff.png)

Reply:
@bjalder26 Thanks for spotting this. I decided to use x and y as the coordinates even though it means that in this particular case, the y argument is given before the x argument.